components/Main/PageContent/index.spec.tsx

```diff
@@ -0,0 +1,8 @@
+import PageContent from '.'
+import { render } from '@testing-library/react'
+
+test('should show title and content', () => {
+  const { getByText } = render(<PageContent title="JS" content="JavaScript is a programming language." />)
+  expect(getByText('JS'))
```

Comment:
Can we separate groups of similar code with blank lines? So a single blank line between the variable declarations and the `expect` statements.